superchain/init.go

```diff
@@ -19,6 +19,13 @@ func init() {
 	}
 	// iterate over superchain-target entries
 	for _, s := range superchainTargets {
+		// TODO Hackily skip sepolia devnet for now. This is because we populate `Implementations` and
+		// other maps based on chain ID, but the Sepolia and Sepolia devnet target have the same chain
+		// ID, meaning the Sepolia devnet data overwrites the Sepolia data. The actual fix is to key off
```

Comment:
Do you mean same L1 chain ID?

Reply:
Yep exactly

Reply:
Could we move this check down to L107? Most of the init function keys by L1 chain name, so could stay in place and we just skip the implementation part?

Reply:
@geoknee Done in 5112ebf